.github/pull_request_template.md

```diff
@@ -1,11 +1,12 @@
 ### What problem do you want to solve?
 
-Reference the issue or discussion. Provide a description of your implementation and the
-changes.
+Reference the issue or discussion, if there is any. Provide a description of your
+implementation and the changes.
 
 ### Todo
 
 - [ ] Target the right branch and pick an appropriate title.
 - [ ] Put `Closes #XXXX` in your comment or `(Closes #XXX)` in your PR title to
-  auto-close the relevant issue.
+      auto-close the relevant issue.
```

Comment:
once the pull request is accepted. This is not applicable if there is no corresponding issue.